.changeset/smart-plants-hide.md

```diff
@@ -0,0 +1,6 @@
+---
+"@layerzerolabs/test-devtools-evm-foundry": major
+"@layerzerolabs/oapp-alt-evm": major
```

Comment:
This will immediatelly bump the version to `1.0.0` which is not what we want I believe

Reply:
just added a minor version for devtools